base/server/etc/default.cfg

```diff
@@ -100,6 +100,13 @@ pki_ca_signing_nickname=caSigningCert cert-%(pki_instance_name)s CA
 pki_external_ca_cert_path=
 pki_ca_signing_cert_path=%(pki_external_ca_cert_path)s
 
+# Separate from pki_ca_signing_cert_path, and for use when a one-shot
+# installation process is used for additional subsystems on a host
+# lacking a CA, pki_ca_cert_path is used to establish trust to an
+# existing CA installation on another system. Specify one or more
+# PKCS#7 blobs in a file.
+pki_ca_cert_path=
```

Comment:
If possible we probably should reuse existing parameters instead of introducing new
ones if they have a similar purpose. Originally, the `pki_external_ca_cert_path` was
used to specify the sub CA cert for sub CA installation where the cert was signed by an
external CA. Now it has been replaced with a more generic `pki_ca_signing_cert_path`
which cover both the sub CA installation and CA migration as well. Basically all
`pki_<tag>_cert_path` can be used to provide the certs for the new subsystem being
installed. I think reusing the `pki_ca_signing_cert_path` would be appropriate.

Reply:
I'm not sure the cert we're importing is necessarily the cert the CA is actively using to sign certificates. In the easiest of cases, it'll be the CA root certificate for the CA instance we're currently installing with. However, it could be that there's an external CA certificate used to sign the `sslserver` cert -- in which case, we'd prefer that external CA certificate. _Additionally_, that external CA might not even have cross-CA trust (i.e., we might not even be a sub CA of that CA). That's why I named the parameter how I did. :) Should I change to using this parameter? Maybe they're used the same way...